hack/common.sh

```
@@ -32,8 +32,8 @@ TESTS_OUT_DIR=$OUT_DIR/tests
 
 function build_func_tests() {
     mkdir -p ${TESTS_OUT_DIR}/
-    ginkgo build ${KUBEVIRT_DIR}/tests
-    mv ${KUBEVIRT_DIR}/tests/tests.test ${TESTS_OUT_DIR}/
+    ginkgo build ${KUBEVIRT_DIR}/tests/kubernetes
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)